modules/controlfreec/1.1/config/freec/config_WGS.txt

```diff
@@ -13,22 +13,44 @@ gemMappabilityFile = referenceFile
 outputDir = OUTDIR
 BedGraphOutput=FALSE
 breakPointThreshold = breakPointValue
+breakPointType = breakPointTypeBe
 coefficientOfVariation = coefVar
-contamination = numCon
+#contamination = numCon
```

Comment:
if it's commented, does this mean it should be removed?

Reply:
I'm not sure why any of the values are commented, won't that prevent them from being ever used?

Reply:
Yes those values are commented out to give way for the other parameters with lower priority. In this case, coefficientOfVariation would dictate window size, and step size relies on window size. If either of those have values, then coefficientOfVariation is overruled, so both windowSize and Step are commented out.

In the case of contamination, if contamination contains a value, it will override contaminationAdjustment's auto-correction. Unless we know the contamination by normal in a given sample, I think letting control-freec decide is the best choice

Reply:
Can you document that briefly in the config? ie. a line below that says something to the effect of:
#uncomment the above line if you want to specify tumour purity, otherwise allow ControlFREE to determine this automatically (recommended)